packages/agents/relayer/README.md

```diff
@@ -0,0 +1,18 @@
+# Relayer
+
+Relayers are a decentralized network of infrastructure operators that can execute smart contract transactions on behalf of a user in exchange for a small fee. Because the last leg of a cross-chain transaction requires execution on the destination domain, relayers play an important role in completing the full flow.
```

Comment:
Nice explanation !
Relayers are used in all off-chain use-cases where we need to "write" to a contract

Reply:
Sure, adding that too